pkg/cfn/manager/cluster.go

```diff
@@ -98,6 +102,10 @@ func (c *StackCollection) AppendNewClusterStackResource(dryRun bool) (bool, erro
 		return false, err
 	}
 
+	if err := printer.LogObj(logger.Debug, "newStack = \\\n%s\n", newStack.Template()); err != nil {
```

Comment:
There is `logger.Debug("newTemplate = %s", newTemplate)` down below, it does the same thing, doesn't it?

Reply:
yes, I was not sure what to do with it, the debug log shows JSON only, I can remove it, no problem.

Reply:
if I understand correctly the code, `PrintObj` will just marshall and indent any object